androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/ManageDataLauncherActivity.java

```diff
@@ -77,6 +79,12 @@
     public static final String ACTION_MANAGE_TRUSTED_WEB_ACTIVITY_DATA =
             "android.support.customtabs.action.ACTION_MANAGE_TRUSTED_WEB_ACTIVITY_DATA";
 
+    public static final String SITE_SETTINGS_SHORTCUT_ID =
+            "android.support.customtabs.action.ACTION_MANAGE_TRUSTED_WEB_ACTIVITY_DATA";
```

Comment:
We should probably change the value of this (ending with something like "SITE_SETTINGS_SHORTCUT"?)